packages/aws-cdk-docs/src/getting-started.rst

```diff
@@ -31,31 +31,32 @@ used to setup the :ref:`credentials <credentials>` for your AWS account.
 Install the command-line toolkit
 --------------------------------
 
-The toolkit can be installed via `npm <https://www.npmjs.org>`_ as follows:
+Install the toolkit using the following `npm <https://www.npmjs.org>`_ command.
 
 .. code-block:: sh
 
     npm install -g aws-cdk
 
-You can run this command to see the currently installed version of the toolkit
-(This guide is aligned with |version|):
+After you've installed the toolkit,
+run the following command to see the currently installed version of the toolkit.

```

Comment:
Since this is not a required step, I think we should say something like "you can" or "if needed" or add some optionality to it.